Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -115,6 +117,9 @@ namespace Multiplayer
     AZ_CVAR(AZ::TimeMs, bg_captureTransportPeriod, AZ::TimeMs{1000}, nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
         "How often in milliseconds to record transport metrics.");
 
+    AZ_CVAR(bool, sv_multithreadedConnectionUpdates, true, nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
```

Comment:
How does the dev control # of threads? Whats the impact of the rest of application performance?

Reply:
By the default the number of threads are set by JobManagerComponent for the entire system and shared among all jobs that use the default Job Context.

Reply:
The exact number depends on the system and the configuration:
```cpp
    void JobManagerComponent::Activate()
...
        int numberOfWorkerThreads = m_numberOfWorkerThreads;
        if (numberOfWorkerThreads <= 0) // spawn default number of threads
        {
        #if (AZ_TRAIT_THREAD_NUM_JOB_MANAGER_WORKER_THREADS)
            numberOfWorkerThreads = AZ_TRAIT_THREAD_NUM_JOB_MANAGER_WORKER_THREADS;
        #else
            uint32_t scaledHardwareThreads = Threading::CalcNumWorkerThreads(cl_jobThreadsConcurrencyRatio, cl_jobThreadsMinNumber, 0, cl_jobThreadsNumReserved);
            numberOfWorkerThreads = AZ::GetMin(static_cast<unsigned int>(desc.m_workerThreads.capacity()), scaledHardwareThreads);
        #endif // (AZ_TRAIT_THREAD_NUM_JOB_MANAGER_WORKER_THREADS)
        }
```